.github/workflows/push_pr.yaml

```diff
@@ -50,17 +50,26 @@ jobs:
               echo "No unit tests found for $chart"
             fi
           done
+      - uses: actions/setup-go@v2
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Check README.md is up to date
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          go install github.com/norwoodj/helm-docs/cmd/helm-docs@v1.11.0
+          make build-chart-docs
+          if [ ! -z "$(git status --porcelain)" ]; then 
+            git diff
+            exit 1
+          fi
```

Comment:
I would move this to other job like codespell or to the static analysis one.

I would want to have the biggest amount of errors in the CI before having to re-run it, i.e.: having an error of spell, linting, and invalid README.

Reply:
you mean to [this](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/.github/workflows/push_pr.yaml#L138) job?

I added here since this is the `chart-lint` job and also is being triggered only if a chart is changed, like the `lint charts` step.

Reply:
take into in account that all tests can be running locally before push by using the make targets.

Reply:
Fair enough.